src/main/java/tech/jhipster/lite/module/domain/JHipsterModulesApplyer.java

```diff
@@ -179,6 +183,18 @@ private JavaBuildCommands buildDependenciesChanges(JHipsterModule module) {
     return module.javaDependencies().buildChanges(javaVersions.get(), projectDependencies.get(module.projectFolder()));
   }
 
+  private JavaBuildCommands buildPropertiesChanges(JHipsterModule module) {
+    return new JavaBuildCommands(
```

Comment:
You should delegate that to `JHipsterModuleBuildProperties`:
```suggestion
    return module.javaBuildProperties().buildChanges();
```

Reply:
We should probably also change `JHipsterModuleJavaBuildProfile.properties` from `Map<PropertyKey, PropertyValue>` to `JHipsterModuleBuildProperties` and call `JHipsterModuleBuildProperties.buildChanges(BuildProfileId)` in `tech.jhipster.lite.module.domain.javabuildprofile.JHipsterModuleJavaBuildProfiles#toAddPropertyCommands`.
So logic of creating `SetBuildProperty` would be centralized in `JHipsterModuleBuildProperties`